src/backend/src/transformers/change-requests.transformer.ts

```diff
@@ -59,6 +65,9 @@ const changeRequestTransformer = (
     leftoverBudget: changeRequest.stageGateChangeRequest?.leftoverBudget ?? undefined,
     confirmDone: changeRequest.stageGateChangeRequest?.confirmDone ?? undefined
   };
+
+  const crStatus = calculateStatus(cr.implementedChanges, cr.accepted, cr.dateReviewed);
+  return { ...cr, status: crStatus };
```

Comment:
this is the only thing that anthony might complain about but i think its fine. the alternate way to do it is to make a const at the top...